pydantic/main.py

```diff
@@ -19,14 +20,26 @@
 from .validators import dict_validator
 
 
+class ExtraAttributes(Enum):
```

Comment:
`IntEnum` please and no need for auto. `1` and `2` should be fine.

That way people aren't forced to import to use this.

Reply:
On reflection, maybe better to use `(str, Enum)` and have values of `allowed`, `ignored` and `forbidden` - should be clearer for people to use.

Reply:
you mean like this?
```python
class Extra(str, Enum):
    allowed = 'allowed'
    ignored = 'ignored'
    forbidden = 'forbidden'
```

Reply:
yes

Reply:
Hmm, how is that exactly used?